tests/ContentServiceTest.php

```diff
@@ -75,24 +79,50 @@ public function it_gets_an_item() : void
 
     /**
      * @test
+     * @backupGlobals enabled
      */
     public function it_adds_an_item() : void
     {
+        $_ENV['ASSETS_ORIGIN'] = '~.+~';
+
         self::bootKernel();
 
+        self::mockApiResponse(
+            new Psr7Request('GET', 'https://www.example.com/new-article/assets/figure1.jpg'),
```

Comment:
what is responding here doesn't look like an API

Reply:
This is the request to fetch the original asset. Can be anywhere.

Reply:
So `mockApiResponse` is too constraining?

Reply:
Ah, bad name through copy/paste.